SECURITY.md

```diff
@@ -0,0 +1,30 @@
+# Security Policy
+
+Think you've found a vulnerability in the Uptane Standard? Please review the following details for responsible disclosure.
+
+## Supported Versions
+
+We release updates to the Uptane specification to address security vulnerabilities. Which versions are eligible to receive updates are listed below:
+
+| Version | Supported          |
```

Comment:
Should we support all released versions?

Reply:
What does "support" mean in this case? IMO it doesn't really make sense to update released versions. We could conceivably patch older releases with bugfixes, but I'd probably rather just focus on updating the latest version.

Reply:
I'm inclined to agree with @pattivacek here that we should fix only the current version, but does this raise any potential backwards compatibility issues? I can't think of any changes to date that would break compatibility, but might it be an issue in the future?

Reply:
I agree, we can probably just say "you may report errata for the most recent standard version" or something.